src/libcore/ops.rs

```diff
@@ -64,6 +64,20 @@
 //!
 //! See the documentation for each trait for a minimum implementation that
 //! prints something to the screen.
+//!
+//! This example shows the behavior of the various `Range*` structs.
+//!
+//! ```rust
+//! let arr = [0, 1, 2, 3, 4];
+//!
+//! assert_eq!(arr[ .. ], [0,1,2,3,4]); // RangeFull
+//! assert_eq!(arr[ ..3], [0,1,2    ]); // RangeTo
+//! assert_eq!(arr[1.. ], [  1,2,3,4]); // RangeFrom
+//! assert_eq!(arr[1..3], [  1,2    ]); // Range
+//!
+//! assert_eq!(arr[ ...3], [0,1,2,3 ]); // RangeToIncusive
+//! assert_eq!(arr[1...3], [  1,2,3 ]); // RangeInclusive
```

Comment:
I know you copied these from the existing examples, but I wonder if we should eliminate the extra spacing, since it's not idiomatic Rust. Hmm